elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -62,7 +62,17 @@ $(function() {
         </ul>
         <ul class="list-group submission_table">
             {{#student_submission}}
-            	<li class="{{block_formatting}}" style="margin-left: {{indent}}px">{{{inner_html}}}</li>
+            	<li class="{{block_formatting}}" style="margin-left: {{indent}}px;">
+                    {{#first-wrong}}
+                    <div class="feedback_badge_container">
+                        <span class="badge {{badge_type}} "><i class="fa {{icon}}" aria-hidden="true"></i></span>
+                        <span>{{{inner_html}}}</span>
+                    </div>
```

Comment:
Is the whole badge container thing necessary? why can't you just drop in the badge in front of the other html? Also, I don't think the name "badge container" makes sense when the content is also going in the container.

Reply:
Using the `d-inline-block` rule, which is how the badge is placed elsewhere works for the default styling but not for code styling. The name was just a placeholder so I'm happy to replace it with something else but the css associated with that rule was the only way I was able to get the badge placement right for both default and code format: 
![image](https://user-images.githubusercontent.com/16738261/188636571-cbcf0c03-96b5-4d84-af04-9ff68826475e.png)
![image](https://user-images.githubusercontent.com/16738261/188636686-b81b1761-546b-435d-aa1f-bea714c6c2c4.png)

Reply:
The above is what happens both when using the existing d-inline-block and when removing the div and just dropping the badge in front.